tests/test_query.py

```diff
@@ -842,12 +844,18 @@ def test__check_data_availability(self,
 
         assert all_available == query._check_data_availability()
 
+    @pytest.mark.parametrize("scenario",
+                             [('search-all-fields'),
+                              ('primary-field-available'),
+                              ],
+                             )
```

Comment:
```suggestion
    @pytest.mark.parametrize(
        "scenario",
        [
            ('search-all-fields'),
            ('primary-field-available'),
        ],
    )
```
A bit easier to read.